Lib/test/test_coroutines.py

```diff
@@ -2411,10 +2413,13 @@ async def corofn():
                 coro_repr = repr(coro)
 
                 # clear reference to the coroutine without awaiting for it
+                coro_repr = repr(coro)
```

Comment:
It is already evaluated three lines above.

Reply:
Oh sorry, I enabled automerge before you posted your review. I wrote https://github.com/python/cpython/pull/129684 to remove the duplicated variable and remove a few other minor similar issues.